src/private/WRLDRouteViewAmalgamationHelper.mm

```diff
@@ -0,0 +1,149 @@
+#import "WRLDRouteViewAmalgamationHelper.h"
+#import "WRLDRouteViewHelper.h"
+
+@implementation WRLDRouteViewAmalgamationHelper
+
++ (NSMutableArray *) CreatePolylines:(const WRLDRoutingPolylineCreateParamsVector&)polylineCreateParams width:(CGFloat)width miterLimit:(CGFloat)miterLimit
+{
+    const auto& ranges = [WRLDRouteViewAmalgamationHelper BuildAmalgamationRanges:polylineCreateParams];
+    
+    NSMutableArray* result = [[NSMutableArray alloc] initWithCapacity:ranges.size()];
+    
+    for (const auto& range : ranges)
+    {
+        WRLDPolyline *polyline = [WRLDRouteViewAmalgamationHelper CreateAmalgamatedPolylineForRange:polylineCreateParams startRange:range.first ndEndRange:range.second width:width miterLimit:miterLimit];
+        if(polyline != nil) {
+            [result addObject:polyline];
+        }
```

Comment:
Would be good to decide which format we want to use and then stick to it. For example,
```
if () {
}
```
vs
```
if ()
{
}
```